### DISCUSSION TITLE
Why QuickLook has the highest network activities?

### DISCUSSION THREAD

**Discussion by estilog** [QL-Win/QuickLook#1300] [General]
![Screenshot 2023-04-26 230340](https://user-images.githubusercontent.com/118219733/234749367-dbd9d9d7-f87d-4d5d-98b4-d96ea0fad59d.png)

Windows 11 -> Task Manager -> App history shows QuickLook has the highest network activities for all my apps and processes. What is it doing in the background?

**Comment by xupefei** [comment]
I don't know either. Perhaps you're previewing a lot of HTML files?
